rustls/Cargo.toml

```
@@ -17,6 +17,36 @@ build = "build.rs"
 [features]
 default = []
 
+# NOTE: This unstable & undocumented feature is intended to help make it easier to configure
+# docs.rs to generate the documentation for all features where this is needed,
+# excluding any potentially problematic features such as:
+# - `unsafe-assume-single-core`
+# - FIPS (if added back to this fork someday in the future)
+#
+# Keeping some other features in this list to help make developer testing with
+# almost all features a little easier.
+#
+# Developer testing with all features enabled except for `unsafe-assume-single-core` can be
+# approximated by: --features unstable-api-all,critical-section
+#
+# IMPORTANT: KEEP THIS FEATURE NAME IN SYNC with the following:
+# - [package.metadata.docs.rs] section further below
+# - docs generate job configured in: .github/workflows/docs.yml
+# - other places such as GitHub CI & admin script(s)
+unstable-api-all = [
+  "aws-lc-rs",
+  "brotli",
+  "custom-provider",
+  "logging",
+  "read_buf",
+  "ring",
+  "prefer-post-quantum", # (also covers "aws-lc-rs" until removed)
```

I think the comment on this line is useless now & should be removed:

```suggestion
  "prefer-post-quantum",
```

---

My original idea was just to make a quick list to help simplify doc generation. My scope has changed a little to help simplify developer testing as well, this is why ~~most~~ *all* features (except for new features added further below) are now in this list.